content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -40,18 +40,21 @@ The following table presents the list of collected features and the minimal Agen
 | Feature | Minimal Agent version | Minimal Cluster Agent version |
 |---|---|---|
 | Container Cost Allocation | 7.27.0 | 1.11.0 |
+| GPU Container Cost Allocation | 7.54.0 | 7.54.0 |
```

Comment:
When you asked the #container-app team were they unable to provide a corresponding cluster agent version? This doesn't seem right to me

Reply:
After 1.11 they changed to using a 7.xx version numbering convention. It didn't seem like they had a corresponding cluster agent version. I discussed with @enemyghost and we decided on using 7.54.0 for both